src/lib/mongoBackend/mongoSubCache.cpp

```diff
@@ -606,6 +606,12 @@ void mongoSubCountersUpdate
 {
   std::string  collection = getSubscribeContextCollectionName(tenant);
 
+  if (subId == "")
+  {
+    LM_E(("Runtime Error (empty subscription id)"));
```

Comment:
I  understand that after the `if (params->registration == false)` ward this is not really needed... However, I like the idea of having it, so in the case of bug and the program flow ends here we have at least a Runtime Error in the logs to signal the problem.

NTC